src/ClientHandle.cpp

```diff
@@ -1860,17 +1867,14 @@ bool cClientHandle::HandleHandshake(const AString & a_Username)
 {
 	if (a_Username.length() > 16)
 	{
-		Kick("Your username is too long(>16 characters)");
+		Kick("Your username is too long (>16 characters)");
 		return false;
 	}
 
-	if (!cRoot::Get()->GetPluginManager()->CallHookHandshake(*this, a_Username))
```

Comment:
The [API document on the hook](https://api.cuberite.org/OnHandshake.html) says that plugins should return true to deny entry. Is this correct?